recipe/php_meal.rb

```diff
@@ -215,6 +219,19 @@ def install_libuv
     )
   end
 
+  def install_argon2
+    %q((
+      cd /tmp
+      curl -L -O https://github.com/P-H-C/phc-winner-argon2/archive/20171227.tar.gz
```

Comment:
Could we find a more official source for this dependency?

Reply:
I looked around and that seemed to be the official github repo.